packages/pattern-picker/src/styles.scss

```
@@ -21,9 +20,19 @@ $patterns: "17-2", "black", "ella-d", "link-cloud", "matt-smith", "ose-maiko", "
 	cursor: pointer;
 	padding: 0;
 	overflow: hidden;
-	height: calc(65vh - 50px);
+	height: calc(100% - 10rem);
 	outline: none;
 
+	@include break-small {
+		height: calc(65vh - 50px);
```

Let's use `svh` whenever we use `vh`. It's important on iOS

```suggestion
		height: calc(65vh - 50px);
		
		@supports ( min-height: 65svh ) {
			min-height: calc(65svh - 50px);
		}
```

---

@alshakero after this push I don't think its necessary to apply that rule everywhere
https://github.com/Automattic/wp-calypso/pull/69061/commits/54126b41c0fddbb53dcb6e3fc47e53f054a1840f